### PR TITLE
Flux bubble should decrease colony building time

### PR DESCRIPTION
**Pull request by MichaelCourtney**

### CODE REVIEW

**Comment by MichaelCourtney** [comment]
Issue #3689

File: default/scripting/buildings/colonies/SP_BANFORO.focs.txt [hunk 1]
```diff
@@ -9,22 +9,23 @@ BuildingType
...
+               And [
+                   Species name = "SP_BANFORO"
+                   Population low = [[MIN_RECOLONIZING_SIZE]]
+                   Happiness low = 5
+                ]
+                And [
+                    HasSpecial name = "EXTINCT_BANFORO_SPECIAL"
+                    Contains Building name = "BLD_XENORESURRECTION_LAB"
                 ]
+            ]
```
Comment (agrrr3) [inline default/scripting/buildings/colonies/SP_BANFORO.focs.txt]: unintended whitespace changes?

**Review by agrrr3 — COMMENTED** [review]
besides that whitespace thingy LGTM

**Comment by agrrr3** [comment]
note i think the AI has no clue that there is a way to speed up colonization, but i also think it does not need to know

File: default/scripting/buildings/colonies/SP_BANFORO.focs.txt (hunk 1 above)
Comment (MichaelCourtney) [reply default/scripting/buildings/colonies/SP_BANFORO.focs.txt]: Yeah. I didn't manually change this file, I just changed the python script that generates it

**Comment by MichaelCourtney** [comment]
Spatial flux hulls are still showing up under the robotic line in the pedia. If I work out how to change that I'll fix that too

File: default/scripting/buildings/colonies/SP_BANFORO.focs.txt (hunk 1 above)
Comment (MichaelCourtney) [reply default/scripting/buildings/colonies/SP_BANFORO.focs.txt]: Checked this. The spacing for the indent of Banforo was edited in master by Geoff over here: https://github.com/freeorion/freeorion/commit/0741af7426f291a1718108d2920725f562d843dc

The spacing for the indent in this commit is consistent with the other extinct species and the spacing generated by col_bld_gen.py

Comment (geoffthemedio) [reply default/scripting/buildings/colonies/SP_BANFORO.focs.txt]: Probably col_bld_gen.py should be adjusted to fix the indenting.

My editing the the .focs.txt file was a bad idea and is indicating as such in the comment at the start of the file. So reverting that change is fine.

**Review by agrrr3 — COMMENTED** [review]
only thing which could be missing is the colony building for exobots, else it is good to go

**Comment by MichaelCourtney** [comment]
> only thing which could be missing is the colony building for exobots, else it is good to go

Looks like exobot colony building isn't affected as it has no source colony, its always 5 turns to build

**Pull request merged**